src/features/components/templates/signup/with-banner/social/index.tsx

```diff
@@ -51,6 +53,12 @@ const social_button_content: SocialButtonContent[] = [
 
 const PublicSignupSocial = () => {
     const handleSocialSignup = (data_provider: TSocialProvider) => {
+        Analytics?.trackEvent('ce_virtual_signup_form', {
+            signup_provider: data_provider,
+            action: 'started',
+            form_source: isBrowser() && window.location.hostname,
```

Comment:
this point is not mandatory since in other places they're not correct. we have to use something like document.referrer or `history.prev` instead